.github/workflows/test.yml

```diff
@@ -68,109 +149,44 @@ jobs:
-        restore-keys: |
-          ${{ runner.os }}-chocolatey-python-
-          ${{ runner.os }}-chocolatey-
-    - name: Install dependencies
-      if: matrix.os == 'windows-latest'
-      shell: powershell
-      run: |
-        choco install winflexbison3 --version 2.5.18.20190508
-    - name: Select build type
-      if: matrix.os == 'windows-latest'
-      shell: powershell
-      run: echo "OPENQL_BUILD_TYPE=Release" >> $GITHUB_ENV
-    - name: Select build type
-      if: matrix.os != 'windows-latest'
-      run: echo "OPENQL_BUILD_TYPE=Debug" >> $GITHUB_ENV
-    - name: Disable unitary decomposition
-      if: matrix.os != 'windows-latest'
-      run: echo "OPENQL_DISABLE_UNITARY=true" >> $GITHUB_ENV
-    - name: Disable unitary decomposition
-      if: matrix.os == 'windows-latest'
-      run: echo "OPENQL_DISABLE_UNITARY=true" >> $env:GITHUB_ENV
-    - name: Build
-      env:
-        NPROCS: 5
-      run: python -m pip install --verbose .
-    - name: Test
-      run: python -m pytest
-
-# NOTE: disabled conda completely because it is INCREDIBLY slow on CI (multiple
-# hours!) and occasionally just breaks the runners entirely after ~6 hours. It
-# also doesn't appear like anyone is actually using Conda within the lab
-# (anymore), as everything goes through pycQED and it is pip-only.
-#
-#  conda:
-#    name: Conda
-#    runs-on: ${{ matrix.os }}
-#    strategy:
-#      fail-fast: false
-#      matrix:
-#        os:
-#        - ubuntu-latest
-#        - macos-latest
-#        #- windows-latest
-#        #- windows-2016
-#    steps:
-#    - uses: actions/checkout@v2
-#      with:
-#        submodules: recursive
-#    - name: Set up conda
-#      uses: conda-incubator/setup-miniconda@v2
-#      with:
-#        auto-update-conda: true
-#        miniconda-version: "latest"
-#        channel-priority: strict
-#        channels: conda-forge
-#        show-channel-urls: true
-#        use-only-tar-bz2: true
-#    - name: Install Windows dependencies
-#      if: matrix.os == 'windows-2016' || matrix.os == 'windows-latest'
-#      run: choco install winflexbison3 --version 2.5.18.20190508
-#    - name: Install conda dependencies
-#      run: conda install conda-build conda-verify -y
-#    - name: Build & test
-#      env:
-#        NPROCS: 5
-#      run: conda build conda-recipe
-#    - name: Install
-#      # This doesn't seem to work on Windows in CI because everything appears
-#      # to build in a non-default environment and conda is broken for that,
-#      # see https://github.com/conda/conda/issues/7758
-#      # Note that conda build already does a test install in a fresh
-#      # environment, so this is a bit redundant anyway.
-#      if: matrix.os != 'windows-2016' && matrix.os != 'windows-latest'
-#      run: conda install openql --use-local
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
```

Comment:
you're removing testing for 4 versions of Python
what's the reasoning?
Personally I'm fine with that, but then do we no longer support Python < 3.11?

Reply:
I think it's fair